configure/files/my.cnf

```diff
@@ -1,114 +1,189 @@
+# Optimized my.cnf configuration for MySQL/MariaSQL
 #
-# The MySQL database server configuration file.
+# by Fotis Evangelou, developer of Engintron (engintron.com)
```

Comment:
This is a great find and a great resource to keep handy for future tuning of pre-prod machines.
Thanks for including the full source.